src/com/facebook/buck/jvm/groovy/GroovycStep.java

```diff
@@ -61,8 +69,8 @@
 
   @Override
   public int execute(ExecutionContext context) throws IOException, InterruptedException {
-
-    ProcessBuilder processBuilder = new ProcessBuilder(createCommand());
+    ImmutableList<String> command = createCommand();
+    ProcessBuilder processBuilder = new ProcessBuilder(command);
```

Comment:
Two things:
* This change looks spurious
* We shouldn't be adding new uses of `ProcessBuilder` in the codebase.  You didn't do this here, but I'm letting you know.  We should be using `ProcessExecutor` or `ListeningProcessExecutor` instead.